src/Console/SshCommand.php

```diff
@@ -49,7 +49,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $host = $this->deployer->hosts->get($input->getArgument('hostname'));
+        $firstHostname = $this->deployer->hosts->getIterator()->current()->getHostname();
```

Comment:
Allright. Maybe short explaination why I came up with this simple feature.

When using `dep ssh` I expected initial that it directly connects me instead of writing `dep ssh user@my-little-pony.de`all the time.